elasticsearch-log/get_logs_elasticsearch.py

```diff
@@ -11,105 +11,177 @@
+                      action="store", dest="homepath", help="Directory to run from")
     (options, args) = parser.parse_args()
 
-    parameters = {}
+    params = dict()
+
     if options.homepath is None:
-        parameters['homepath'] = os.getcwd()
-    else:
-        parameters['homepath'] = options.homepath
+        params['homepath'] = os.getcwd()
     if options.serverUrl is None:
-        parameters['serverUrl'] = 'https://app.insightfinder.com'
+        params['serverUrl'] = 'http://stg.insightfinder.com'
     else:
-        parameters['serverUrl'] = options.serverUrl
-    return parameters
+        params['serverUrl'] = options.serverUrl
 
+    params['logLevel'] = logging.INFO
+    if options.logLevel == '0':
+        params['logLevel'] = logging.WARNING
+    elif options.logLevel == '1':
+        params['logLevel'] = logging.INFO
+    elif options.logLevel >= '2':
+        params['logLevel'] = logging.DEBUG
 
-def getElasticConfigVars():
-    reportingConfigVars = {}
-    with open(os.path.join(parameters['homepath'], "elasticsearch-log", "config.json"), 'r') as f:
-        config = json.load(f)
-    reportingConfigVars['elasticsearchIndex'] = str(config['elasticsearchIndex'])
-    reportingConfigVars['elasticsearchHost'] = str(config['elasticsearchHost'])
-    reportingConfigVars['elasticsearchPort'] = int(config['elasticsearchPort'])
-    reportingConfigVars['timeFieldName'] = str(config['timeFieldName'])
-    reportingConfigVars['isTimestamp'] = bool(config['isTimestamp'])
-    reportingConfigVars['hostNameField'] = str(config['hostNameField'])
-    if not reportingConfigVars['isTimestamp']:
-        reportingConfigVars['dateFormatES'] = str(config['dateFormatInJodaTime'])
-        reportingConfigVars['dateFormatPython'] = str(config['dateFormatInStrptime'])
-        if 'dataTimeZone' in config and 'localTimeZone' in config:
-            reportingConfigVars['dataTimeZone'] = str(config['dataTimeZone'])
-            reportingConfigVars['localTimeZone'] = str(config['localTimeZone'])
-    return reportingConfigVars
+    return params
 
-def getReportingConfigVars():
-    reportingConfigVars = {}
-    with open(os.path.join(parameters['homepath'], "reporting_config.json"), 'r') as f:
-        config = json.load(f)
-    reporting_interval_string = config['reporting_interval']
-    if reporting_interval_string[-1:] == 's':
-        reporting_interval = float(config['reporting_interval'][:-1])
-        reportingConfigVars['reporting_interval'] = float(reporting_interval / 60.0)
-    else:
-        reportingConfigVars['reporting_interval'] = int(config['reporting_interval'])
-        reportingConfigVars['keep_file_days'] = int(config['keep_file_days'])
-        reportingConfigVars['prev_endtime'] = config['prev_endtime']
-        reportingConfigVars['deltaFields'] = config['delta_fields']
-    return reportingConfigVars
 
+def time_converter(time_string):
+    utc_time = datetime.strptime(time_string, "%Y-%m-%dT%H:%M:%S.%fZ")
```

Comment:
use time format passed in config?

Reply:
Logs pass the timestamp value as a string and in a different format which is why I cannot use the format from config.